homeassistant/components/fritz/strings.json

```diff
@@ -52,5 +52,35 @@
         }
       }
     }
+  },
+  "entity": {
+    "sensor": {
+      "external_ip": { "name": "External IP" },
+      "external_ipv6": { "name": "External IPv6" },
+      "device_uptime": { "name": "Device Uptime" },
```

Comment:
IMO `Last restart` is more human-readable.
![obraz](https://user-images.githubusercontent.com/478555/229477539-c3873771-4ff1-4040-89e9-5935c60c7b8d.png)